src/pages/home/report/ReportActionCompose.js

```diff
@@ -372,9 +369,9 @@ class ReportActionCompose extends React.Component {
      * @param {String} newComment
      */
     updateComment(newComment) {
-        this.textInput.setNativeProps({text: newComment});
         this.setState({
             isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
+            value: newComment,
```

Comment:
We should add `value` to [state](https://github.com/Expensify/App/blob/59019d53723163e99b5ddeb1968dd53a2f1b433f/src/pages/home/report/ReportActionCompose.js#L136-L147)